e2e_tests/test_administration_checklist_page.py

```diff
@@ -15,11 +15,73 @@ def test_should_show_the_administration_assessment_and_the_checklist(
 
     expect(
         page.locator(
-            'td:text("Gibt es ein Monitoring von Kimaschutzmaßnahmen?") + td'
```

Comment:
Will not do anything here, since #295 is based on it. We can have a look at the tests over there.